src/main/java/com/napier/sem/Record.java

```diff
@@ -123,15 +122,35 @@ public Record(String name, String country, String district, long population)
         recordType = App.CITY;
     }
 
+    //Population Residence Report constructor, just used for testing purposes.
+    public Record(String name, Long population, Long populationLivingInCities, Double percentageLivingInCities, Long populationNotLivingInCities, Double percentageNotLivingInCities)
```

Comment:
we have titlecase `Long` here where we have lowercase `long` everywhere else. java's so confusing